chat/setup.sh

```diff
@@ -0,0 +1,6 @@
+git clone https://github.com/abetlen/llama-cpp-python --no-checkout --recurse-submodules
+cd llama-cpp-python
+git checkout v0.3.2
+git submodule update
+curl -O -L https://github.com/abetlen/llama-cpp-python/pull/1552.diff
```

Comment:
this PR is not merged
I think that you should bring the patch here as the PR may change one day to another so you'll never be able to see what will be used.

Or it means we wait until the PR is merged

Reply:
Using rebase on the last commit of the PR instead of the PR diff for better stability